demos/common/cpp_gapi/utils_gapi/include/utils_gapi/ov_backend.hpp

```diff
@@ -32,10 +32,25 @@ struct BackendApplicator<ExecNetwork,
                 config.deviceName                    // device specifier
             }.cfgNumRequests(config.maxAsyncRequests)
             .cfgPluginConfig(config.getLegacyConfig());
-        // E#96606
-        if (!config.mean_values.empty() || !config.scale_values.empty()) {
-            throw std::runtime_error(backend.name + " backend doesn't support neither `mean_values` nor `scale_values` at the moment");
+            
+        if (!config.mean_values.empty() && !config.scale_values.empty()) {
+            std::vector<float> means;
+            split(config.mean_values, ' ', means);
```

Comment:
I don't think we need to parse `string` into `vector<float>` in every backend we have, it should be generic.
Let `BackendsConfig` holds `vector<float>` instead of `string` and transform `string` to `vector` somewhere on application side (main.cpp).

https://github.com/openvinotoolkit/open_model_zoo/blob/fa567dd88238d975d8b2b39032147922137b71ad/demos/common/cpp_gapi/utils_gapi/include/utils_gapi/backend_description.hpp#L27-L33

Reply:
So here I should change BackendsConfig, so that they hold vector values instead of strings. And the conversion of string to vector will happen at the application side.